app/user.go

```diff
@@ -321,12 +321,14 @@ func (a *App) createUserOrGuest(c request.CTX, user *model.User, guest bool) (*m
 	// table in CWS. This is then used to calculate how much the customers have to pay in addition for the extra users. If the
 	// workspace is currently on a monthly plan, then this function will not do anything.
 
-	go func() {
-		_, err := a.SendSubscriptionHistoryEvent(ruser.Id)
-		if err != nil {
-			c.Logger().Error("Failed to create/update the SubscriptionHistoryEvent", mlog.Err(err))
-		}
-	}()
+	if a.Channels().License().IsCloud() {
+		go func(userId string) {
```

Comment:
Nit: we prefer `userID` over `userId`: https://developers.mattermost.com/contribute/more-info/server/style-guide/#hahahugoshortcode-s8-hbhb